src/lib/y2storage/volume_specification.rb

```diff
@@ -155,6 +155,8 @@ def initialize(volume_features)
     def apply_defaults
       @max_size ||= DiskSize.unlimited
       @max_size_lvm ||= DiskSize.zero
+      @proposed ||= true
+      @proposed_configurable ||= true
```

Comment:
I think we should update the documentation to indicate the default values for these settings.

Reply:
Yes, not only for these.